CompetitionTemplate/src/competition/subsystems/vision/InertJetsonServer.java

```diff
@@ -0,0 +1,34 @@
+package competition.subsystems.vision;
+
+import java.util.function.Consumer;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+public class InertJetsonServer implements JetsonServer {
```

Comment:
Do you think that it should clear the last vision data snapshot when the connection becomes unhealthy (to prevent consumers from using old data), or should it keep the data and trust that any consumers will take the lack of recent data into account?